taskcluster/rb_taskgraph/routes.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import time
+
+from taskgraph.transforms.task import index_builder
+
+RAPTOR_ROUTE_TEMPLATES = [
+    "index.{trust-domain}.{project}.v2.branch.{head_ref}.revision.{head_rev}.{build-type}.{abi}",
+    "index.{trust-domain}.{project}.v2.branch.{head_ref}.latest.{build-type}.{abi}",
+    "index.{trust-domain}.{project}.v2.branch.{head_ref}.pushdate.{build_date}.revision.{head_rev}.{build-type}.{abi}",
+    "index.{trust-domain}.{project}.v2.branch.{head_ref}.pushdate.{build_date}.latest.{build-type}.{abi}",
+]
+
+
+@index_builder("raptor")
+def add_raptor_indexes(config, task):
```

Comment:
When is this function called? I don't see any references to `rb_taskgraph.routes` :thinking: